packages/split-view/src/split-view.css

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import './spectrum-split-view.css';
+
+:host {
+    --spectrum-split-view-first-pane-size: 50%;
+}
+
+::slotted(*) {
+    overflow: auto;
+}
+
+::slotted(:first-child) {
+    order: 1;
+}
+
+:host(:not([vertical])) ::slotted(:first-child) {
+    width: var(--spectrum-split-view-first-pane-size);
+}
+
+:host([vertical]) ::slotted(:first-child) {
+    height: var(--spectrum-split-view-first-pane-size);
+}
+
+::slotted(:nth-child(2)) {
+    order: 3;
+    flex: 1;
+}
+
+::slotted(:nth-child(n + 3)) {
+    display: none;
+}
+
+#splitter {
+    order: 2;
+    height: auto; /* for horizontal splitviews without proper outter height value */
+}
+
+:host([vertical][resizable]) #splitter {
+    padding: var(--spectrum-global-dimension-static-size-125) 0;
+    margin: calc(-1 * var(--spectrum-global-dimension-static-size-125)) 0;
+    background-clip: content-box;
+    cursor: ns-resize;
+}
+
+:host(:not([vertical])[resizable]) #splitter {
```

Comment:
Not sure if it's a nit or a good thing, but if you reverse the order of these rules, you should be able to forgo the `:not()` modifier/complexity by assuming the "horizontal" layout is the default?

```
host([resizable]) #splitter { ... }
:host([vertical][resizable]) #splitter { ... }
```

If that's helpful, should apply to the rest of the rules in this file, as well.

This file also gets process for nested selectors (though editor linting might not like it 🤔 ), if you felt approaching this in that way would make it easier to read/manage long term.

I'll open a conversation with the CSS team about getting them interested in adopting these changes so we can hopefully move them out of our code all together, too.

Reply:
Found a better solution here.